other-docs/guides/upgrading/v2.md

```diff
@@ -0,0 +1,103 @@
+
+
+### Publication Checklist (Workflow module)
+
+Altis now includes a Publication Checklist feature, allowing you to ensure conditions are met before publishing. This is fully customisable for your workflow.
+
+The Publication Checklist feature is enabled out-of-the-box, but has no effect until you add checks. Consult the [Publication Checklist documentation](docs://workflow/publication-checklist.md) to discover how to create checks.
+
+
+## Other User Features
+
+### Google Site Verification (SEO module)
+
+You can now manage your site verification with the Google Search Console without requiring codebase changes. See the [documentation on how to configure this](docs://seo/google-site-verification.md).
+
+
+## Other Developer Features and APIs
+
+### Autoloader Helper (Core module)
+
+An autoloader helper is now included in Altis to simplify autoloading WordPress-style file names (e.g. `class-foo.php`). See the [documentation for `Altis\register_class_path()`](docs://core/#autoloader).
+
+
+### AWS SDK configuration (Core module)
+
+AWS credentials can now be configured via the configuration system rather than via constants. See the [documentation for the `modules.core.aws` configuration](docs://core/#aws-sdk).
+
+
+### Developer Tools is now enabled more widely (Developer Tools module)
+
+The Developer Tools module is now enabled for all non-production environments. This includes staging and development environments running on Altis Cloud, in addition to local environments where it was previously enabled.
+
+To revert to the previous behaviour, [use environment-specific configuration](docs://dev-tools/#activating-in-other-environments) to disable on all environments except `local`:
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"dev-tools": {
+					"enabled": false
+				}
+			},
+			"environments": {
+				"local": {
+					"modules": {
+						"dev-tools": {
+							"enabled": true
+						}
+					}
+				}
+			}
+		}
+	}
+}
+```
+
+### Local Chassis improvements (Local Chassis module)
+
+Local Chassis now generates a self-signed SSL certificate to allow testing locally with HTTPS. This certificate needs to be installed on your system and on some browsers; consult the [HTTPS documentation](docs://local-chassis/#using-https-locally) for more information.
+
+If you have an existing Local Chassis install, a manual step is required to update. Run the following from your project directory:
+
+```sh
+cd chassis/extensions/platform-chassis-extension
+git pull
+vagrant provision
+```
+
+Additionally, new convenience commands are available, including `composer chassis shell`. See the [command documentation](docs://local-chassis/#available-commands) for full information.
```

Comment:
We might want to mentioned that Chassis doesn't support Analytics / Experiment results yet.

Reply:
I should probably do that, think it's easy enough to add docker stuff via puppet.